docs/invoice-spec.md

```diff
@@ -43,6 +43,7 @@ The above bindle declares its `bindle` description, and then declares a manifest
 
 - `bindleVersion` is required, and should be `1.0.0` for this version of the specification.
 - `yanked` is a boolean field that indicates whether a Bindle has been yanked. This field appears outside of the `bindle` because it is mutable, though it can only be toggled on. Once set to true, a Bindle MUST NOT be un-yanked. A yanked bundle should never be served in an index or search, but MAY be accessed directly.
+- `yanked_reason` (OPTIONAL) is a string field in which a human-readable reason can be given for yanking the invoice.
```

Comment:
There is inconsistency between the cases for `bindleVersion` and `yanked_reason`.

Reply:
Oh yeah! we changed `bindleVersion` to `bindle_version` a few PRs ago and I must have missed several spots in the docs. Thanks.

Reply:
Fixed throughout docs.